themes/theGarbz/Focus/README.md

```diff
@@ -0,0 +1,54 @@
+# Focus theme v0.1
+
+This theme is based on the classy NOX theme by NoxHirsch and has been modified to Focus on highlighting the most important information for a user. The goal was to create a theme that could easily be read from a distance. Some of the features are listed below:
```

Comment:
Might be nice to link the NOX theme here I think.

Reply:
Agreed and done. Commit 48ac302